### ISSUE TITLE
Accessibility issue with help pages.

### ISSUE THREAD

**Issue by Igor-Ino** [LibraryOfCongress/concordia#1551]
**What behavior did you observe? Please describe the bug**
A clear and concise description of what you experienced.

**How can we reproduce the bug?**

1. Go to 'https://crowd-test.loc.gov/help-center/how-to-transcribe/'
2. Run WAVE web accessibility evaluation tool (Only FF or Chrome)
3. Click at View detail on Left side menu
4. See details about contrast error: <a class="nav-link active" href="/help-center/how-to-review/" style="color: rgb(240, 81, 41); background-color: rgb(255, 255, 255);">
How to review
</a>

**What is the expected behavior?**
There should be no error in help pages

**Got screenshots? This helps us identify the issue**
Add screenshots to help explain your problem.
<img width="701" alt="Screen Shot 2021-09-29 at 21 12 22" src="https://user-images.githubusercontent.com/80710853/135369424-7f018391-b702-47d0-b2c6-68baf7bb6b16.png">

**Desktop (please complete the following information):**

-   OS: [e.g. iOS]
-   Browser [e.g. chrome, safari, FF]

**Additional context**

**Comment by jkueloc** [comment]
Moved to Jira: [CONCD-14](https://staff.loc.gov/tasks/browse/CONCD-14)

**Comment by jkueloc** [comment]
This issue is closed as the work has been moved to the Library's Jira repo as noted above.
